iprange/sortip.go

```diff
@@ -0,0 +1,36 @@
+package iprange
+
+import "net"
+
+// Asc implements sorting in ascending order for IP addresses
+type asc []net.IP
+
+func (a asc) Len() int {
+	return len(a)
+}
+
+func (a asc) Swap(i, j int) {
+	a[i], a[j] = a[j], a[i]
+}
+
+func (a asc) Less(i, j int) bool {
```

Comment:
Absolutely. I assumed we only needed IPv4 support, since the parser is IPv4-only, but future-proofing is always a great idea.